### PR TITLE
docs: fix translation typos and improve quality

### PR DESCRIPTION
**Pull request by mitchell-cheng**
### Description
- Corrected Chinese translation typos
- Reorganized and retranslated some texts to improve readability

### Checklist
- [x] I have ensured this PR does not contain any breaking changes for existing functionality;
- [ ] I have added automated tests that cover the new behavior and removed obsolete tests and code;

### Tested On
- [ ] Emulator
- [x] Real device

### CODE REVIEW

File: core/localization/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -396,23 +399,23 @@
...
     <string name="in_paint_area_whole">整张图片</string>
-    <string name="in_paint_area_only_masked">仅遮罩</string>
+    <string name="in_paint_area_only_masked">仅掩码</string>
 
-    <string name="donate_title_thanks">感谢这些了不起的人的支持❤</string>
+    <string name="donate_title_thanks">感谢这些了不起的人们提供的支持 ❤</string>
     <string name="donate_bs_title">SDAI 是一款 Android 应用程序，它：</string>
-    <string name="donate_bs_sub_title">• 通过稳定的扩散 AI 为您提供创建图像的能力；\n• 让您自由选择生成提供商；\n• 没有广告、遥测并且不会监视您；\n• 是一个开源软件；\n• 欢迎您免费使用。</string>
-    <string name="donate_bs_ending">如果您发现该软件有价值，并且想要表示感谢并表示一点支持，请点击此处的按钮。</string>
+    <string name="donate_bs_sub_title">• 赋予您使用 Stable Diffusion AI 创建图像的能力； • 让您自由选择生成供应商； • 无广告、无遥测，无监控； • 是一款开源软件； • 欢迎您免费使用。</string>
```
Comment (ShiftHackZ) [inline core/localization/src/main/res/values-zh/strings.xml:415]: This string should contain new line characters `\n` after each phrase before every next `•` symbol.

```suggestion
    <string name="donate_bs_sub_title">• 赋予您使用 Stable Diffusion AI 创建图像的能力；\n• 让您自由选择生成供应商；\n• 无广告、无遥测，无监控；\n• 是一款开源软件；\n• 欢迎您免费使用。</string>
```

**Review by ComicSAS — APPROVED** [review]
Looks fine.

Thank you for your contribution!

File: core/localization/src/main/res/values-zh/strings.xml (hunk 1 above)
Comment (ComicSAS) [reply core/localization/src/main/res/values-zh/strings.xml:415]: Right, missed this one.

File: core/localization/src/main/res/values-zh/strings.xml [hunk 2]
```diff
@@ -70,107 +70,110 @@
...
     <!-- 服务器设置提示 -->
-    <string name="hint_server_setup_title">提供您的Stable Diffusion WebUI URL</string>
-    <string name="hint_server_setup_sub_title">使用 Gradio 库实现的稳定扩散的 Web 界面。</string>
-    <string name="hint_valid_urls">以下是服务器URL的示例：\n• http://192.168.0.2:%1$s\n• http://yourdomain.com:%1$s\n• https://yourdomain.com</string>
-    <string name="hint_demo_mode">此模式允许您测试应用程序的行为，即使您没有Stable Diffusion WebUI服务器。\n\n在演示模式下，应用程序忽略用户提示，不使用AI服务器，并返回一些模拟图像。</string>
-    <string name="hint_args_warning" tools:ignore="TypographyDashes">在连接之前确保：\n• 您正在运行AUTOMATIC1111 WebUI并带有标志 --api --listen\n• 您的防火墙没有阻止7860端口\n• 手机与您的PC在同一WiFi下</string>
-    <string name="hint_args_swarm_ui_warning" tools:ignore="TypographyDashes">在连接之前确保：\n• 您的防火墙没有阻止7801端口\n• 手机与您的PC在同一WiFi下</string>
+    <string name="hint_server_setup_title">请提供您的 Stable Diffusion WebUI URL</string>
+    <string name="hint_server_setup_sub_title">使用 Gradio 库实现的 Stable Diffusion 的 Web 界面。</string>
+    <string name="hint_valid_urls">以下是服务器 URL 的示例： • http://192.168.0.2:%1$s • http://yourdomain.com:%1$s • https://yourdomain.com</string>
```
Comment (ShiftHackZ) [inline core/localization/src/main/res/values-zh/strings.xml:136]:
```suggestion
    <string name="hint_valid_urls">以下是服务器 URL 的示例：\n• http://192.168.0.2:%1$s\n• http://yourdomain.com:%1$s\n• https://yourdomain.com</string>
```

**Review by ComicSAS — APPROVED** [review]
Fixed.

File: core/localization/src/main/res/values-zh/strings.xml [hunk 3]
```diff
@@ -70,107 +70,110 @@
...
-    <string name="hint_server_setup_sub_title">使用 Gradio 库实现的稳定扩散的 Web 界面。</string>
-    <string name="hint_valid_urls">以下是服务器URL的示例：\n• http://192.168.0.2:%1$s\n• http://yourdomain.com:%1$s\n• https://yourdomain.com</string>
-    <string name="hint_demo_mode">此模式允许您测试应用程序的行为，即使您没有Stable Diffusion WebUI服务器。\n\n在演示模式下，应用程序忽略用户提示，不使用AI服务器，并返回一些模拟图像。</string>
-    <string name="hint_args_warning" tools:ignore="TypographyDashes">在连接之前确保：\n• 您正在运行AUTOMATIC1111 WebUI并带有标志 --api --listen\n• 您的防火墙没有阻止7860端口\n• 手机与您的PC在同一WiFi下</string>
-    <string name="hint_args_swarm_ui_warning" tools:ignore="TypographyDashes">在连接之前确保：\n• 您的防火墙没有阻止7801端口\n• 手机与您的PC在同一WiFi下</string>
+    <string name="hint_server_setup_title">请提供您的 Stable Diffusion WebUI URL</string>
+    <string name="hint_server_setup_sub_title">使用 Gradio 库实现的 Stable Diffusion 的 Web 界面。</string>
+    <string name="hint_valid_urls">以下是服务器 URL 的示例：\n• http://192.168.0.2:%1$s\n• http://yourdomain.com:%1$s\n• https://yourdomain.com</string>
+    <string name="hint_demo_mode">此模式允许您测试应用程序的行为，即使您没有 Stable Diffusion WebUI 服务器  在演示模式下，应用程序将忽略用户 prompt，不使用 AI 服务器，并返回一些模拟图像。</string>
```
Comment (ShiftHackZ) [inline core/localization/src/main/res/values-zh/strings.xml:137]: Maybe it's worth bringing `\n\n` back.

File: core/localization/src/main/res/values-zh/strings.xml [hunk 4]
```diff
@@ -70,107 +70,110 @@
...
-    <string name="hint_swarm_ui_sub_title">模块化稳定扩散 Web 用户界面，重点在于使工具易于访问、高性能和可扩展性。</string>
+    <string name="hint_swarm_ui_sub_title">模块化的 Stable Diffusion Web 用户界面，专注于易访问、高性能和可扩展性。</string>
 
     <!-- 本地扩散提示 -->
-    <string name="hint_local_diffusion_title" translatable="false">本地扩散</string>
-    <string name="hint_local_diffusion_sub_title">此配置使用 Microsoft ONNX 运行时，并允许在手机上运行稳定的 Diffusion AI 生成，无需连接到远程服务器/云。</string>
-    <string name="hint_local_diffusion_warning">警告！本地扩散功能处于测试版。不要期望使用本地模式获得高质量图像。 \n\n此实现可能在不强大的手机上运行不佳。生成性能和速度取决于您的手机资源（CPU、RAM）和生成的图像大小（图像越小，生成越快）。</string>
+    <string name="hint_local_diffusion_title" translatable="false">Local Diffusion Microsoft ONNX</string>
+    <string name="hint_local_diffusion_sub_title">此配置使用 Microsoft ONNX 运行时，您可以在手机端运行 Stable Diffusion AI 的生成功能，而无需连接至远程服务器/云。</string>
+    <string name="hint_local_diffusion_warning">警告！Local Diffusion 功能处于测试版。可能无法获得高质量图像。 此功能在资源受限设备上可能不及预期。生成性能和速度取决于您的手机资源（CPU、RAM）和生成的图像大小（图像越小，生成越快）。</string>
```
Comment (ShiftHackZ) [inline core/localization/src/main/res/values-zh/strings.xml:174]: Maybe it's worth bringing `\n\n` back.

**Review by ComicSAS — APPROVED** [review]
Next time, please be careful when removing "\n" characters from translation strings.

**Review by ShiftHackZ — APPROVED** [review]
Checked on my device, looks good to me. 
Thanks!

**Pull request merged**